src/quo2/components/switchers/group_messaging_card/view.cljs

```diff
@@ -0,0 +1,37 @@
+(ns quo2.components.switchers.group-messaging-card.view
+  (:require
+    [react-native.core :as rn]
+    [quo2.components.markdown.text :as text]
+    [quo2.components.avatars.group-avatar :as group-avatar]
+    [quo2.components.switchers.base-card.view :as base-card]
+    [quo2.components.switchers.style :as style]
```

Comment:
I think we should avoid reusing styles in many components - either styles are encapsulated in a component and that component is reused or we just duplicate and put these styles in each component 👍

Reply:
e.g make a component - `card-title` with those styles added and expose that component instead